README.md

```diff
@@ -244,6 +244,29 @@ rules:
   verbs: ["watch","list","get","update"]
 ```
 
+Make sure to set `StatefulSet.metadata.name` and `StatefulSet.spec.serviceName` to the same value;
+otherwise, autoscaling will not trigger (`mc-router.itzg.me/externalServerName` annotation is also required for it to work,
+however if you are not using a DNS you can specify an IP address as well `"mc-router.itzg.me/externalServerName": "1.2.3.4"`):
```

Comment:
Please remove the part that's in parentheses. It's already documented above in the kubernetes section. You can add the info about using an IP address there.

Reply:
Makes sense, just removed it now :)